test/parallel/test-crypto.js

```diff
@@ -54,19 +54,18 @@ assert.throws(function() {
 
 
 function assertSorted(list) {
-  // Array#sort() modifies the list in place so make a copy.
-  const sorted = list.slice().sort();
-  assert.deepStrictEqual(list, sorted);
+  // list is sorted and contains no duplicates
+  assert(list.every(
+    (val, index, array) => index === array.length - 1 || val < array[index + 1]
+  ));
 }
 
 // Assume that we have at least AES-128-CBC.
-assert.notStrictEqual(0, crypto.getCiphers().length);
```

Comment:
That is reasonably compelling, though an extra assert to be explicit about the condition isn't bad. As you wish, I'm OK either way.